src/DetailsView/reports/assessment-report.scss

```diff
@@ -8,6 +8,8 @@
 @import './components/assessment-summary-details.scss';
 @import './components/assessment-scan-details.scss';
 @import './components/assessment-report-body-header.scss';
+@import './components/automated-check-issue-details-card.scss';
```

Comment:
At first glance it looks odd to have automated checks styling here

Reply:
Yes. it was a mistake. removed.